src/gluonts/model/deepstate/_network.py

```diff
@@ -31,6 +32,7 @@ def __init__(
         num_layers: int,
         num_cells: int,
         cell_type: str,
+        output_transform: Optional[ComposedBijectionHybridBlock],
```

Comment:
Same as for the `DeepStateEstimator`: couldn't one use whatever `Bijection` here in principle?